spec/controllers/searchgov_urls_controller_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'spec_helper'
+
+describe Admin::SearchgovUrlsController do
+  fixtures :users
+
+  let(:admin) { users(:affiliate_admin) }
+  let(:searchgov_url) { SearchgovUrl.create(url: 'agency.gov/test') }
```

Comment:
Try to use bang methods in your tests, to ensure that your setup steps don't fail silently: `SearchgovUrl.create!(url: 'agency.gov/test')`